src/clr-angular/utils/_components.clarity.scss

```diff
@@ -98,14 +98,25 @@
 @import "../popover/tooltip/tooltips.clarity";
 
 //Forms
-@import "../forms-deprecated/utils/form-variables.clarity";
-@import "../forms-deprecated/utils/mixins.clarity";
-@import "../forms-deprecated/inputs.clarity"; // depends on variables.clarity, mixins.clarity, color.clarity, helpers.clarity
-@import "../forms-deprecated/checkbox/checkbox.clarity"; // depends on variables.clarity, mixins.clarity, color.clarity, helpers.clarity
-@import "../forms-deprecated/radio.clarity"; // depends on variables.clarity, mixins.clarity, color.clarity, helpers.clarity
-@import "../forms-deprecated/select.clarity"; // depends on variables.clarity, mixins.clarity, color.clarity, helpers.clarity, layers.clarity, icons.clarity, images.clarity
-@import "../forms/datepicker/datepicker.clarity";
-@import "../forms-deprecated/forms.clarity"; // depends on variables.clarity, mixins.clarity, color.clarity, helpers.clarity
+@import "../forms/styles/mixins.clarity";
+@import "../forms/styles/containers.clarity";
+@import "../forms/styles/form.clarity";
+@import "../forms/styles/checkbox.clarity";
+@import "../forms/styles/file.clarity";
+@import "../forms/styles/input.clarity";
+@import "../forms/styles/input-group.clarity";
+@import "../forms/styles/radio.clarity";
+@import "../forms/styles/textarea.clarity";
+
+//Forms Deprecated
+// @import "../forms-deprecated/utils/form-variables.clarity";
```

Comment:
Why are we commenting this out? If we are going to merge this into master, we want the old forms to work too right?